scipy/stats/_hypotests.py

```diff
@@ -132,6 +134,242 @@ def epps_singleton_2samp(x, y, t=(0.4, 0.8)):
     return Epps_Singleton_2sampResult(w, p)
 
 
+class CramerVonMisesResult:
+    def __init__(self, statistic, pvalue):
+        self.statistic = statistic
+        self.pvalue = pvalue
+
```

Comment:
It would be nice to include a `__repr__` method:
```suggestion

    def __repr__(self):
        return (f"{self.__class__.__name__}(statistic={self.statistic}, "
                f"pvalue={self.pvalue})")
```

Reply:
I added the `__repr__` method, and the tests are running again.  If the tests pass, I'll merge the PR.